textdb/textdb-storage/src/main/java/edu/uci/ics/textdb/storage/DataReaderPredicate.java

```diff
@@ -0,0 +1,27 @@
+package edu.uci.ics.textdb.storage;
+
+import org.apache.lucene.search.Query;
+
+import edu.uci.ics.textdb.api.common.IPredicate;
+import edu.uci.ics.textdb.api.storage.IDataStore;
+
+/**
+ * Created by sandeepreddy602 on 05-06-2016.
+ */
+public class DataReaderPredicate implements IPredicate {
+    private IDataStore dataStore;
+    private Query query;

```

Comment:
`Query` is Lucene class (`org.apache.lucene.search.Query`). We can't have an interface here.